### PR TITLE
feat: support event logs extraction from the opcode logger

### PR DESCRIPTION
**Pull request by lljxx1**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,11 +8,17 @@
   "types": "dist/index.d.ts",
   "scripts": {
     "build": "tsc",
+    "test": "jest --no-watchman",
     "clean": "rm -rf dist tsconfig.tsbuildinfo",
     "prepublishOnly": "yarn clean && yarn build"
   },
   "dependencies": {
+    "@ethereumjs/util": "^9.0.0",
```
Comment (georgeroman) [inline package.json]: I see we only use `bytesToHex` from this package. Isn't a similar method available in `ethers`? Would avoid adding a new package just for a single basic method.

File: package.json [hunk 2]
```diff
@@ -8,11 +8,17 @@
...
+    "ts-jest": "^29.0.1",
```
Comment (georgeroman) [inline package.json]: `ts-jest`, `ts-node`, `@types/jest` and `jest` should all be `devDependencies` - they shouldn't be included in the build.

**Review by georgeroman — CHANGES_REQUESTED** [review]

**Comment by georgeroman** [comment]
@lljxx1 What do you think of making on-chain calls to get the test fixtures? Feels a bit extreme to save those million lines traces in the code when we can easily fetch them via `debug_traceTransaction` method on-demand.

**Comment by lljxx1** [comment]
@georgeroman I filtered out the unused opcodes, and the file size was down a little bit.

**Pull request merged**